tests/unit/frameworks/continuous_learning/multihead/multihead_test.py

```diff
@@ -78,18 +98,45 @@ def test_class_scenario(self):
         Test all target values to be changing based on the task number in the class
         scenario.
         """
-        train_loaders, test_loaders = setup("splitMNIST", "class")
+        train_loaders, test_loaders = setup("class")
         for task_num, train_loader in enumerate(train_loaders):
             min_target, max_target = 2 * task_num, 2 * task_num + 1
             for _data, target in train_loader:
-                self.assertLess(target.max().item(), max_target + 1)
-                self.assertGreaterEqual(target.min().item(), min_target)
+                # print(target, min_target, max_target)
+                try:
+                    self.assertLess(target.max().item(), max_target + 1)
+                    self.assertGreaterEqual(target.min().item(), min_target)
+                except AssertionError:
```

Comment:
Why are you ignoring the `AssertionError`?

Reply:
oops, I wrote this while debugging, I will fix it now.